pages/api/submissions/index.ts

```diff
@@ -0,0 +1,29 @@
+import forms from 'data/flexibleForms/forms';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { addCase } from 'lib/cases';
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  if (req.method === 'POST') {
+    const { socialCareId, formId } = req.body;
+
+    // TODO: start a new submission here
+    // const newSubmission = await addCase({
+    //   socialCareId,
+    //   formId,
+    // });
+
+    // TODO: send back new case object
```

Comment:
I really don't think we want commented sections of code.

https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Reply:
so be it